src/core/Table/Table.tsx

```diff
@@ -231,6 +232,11 @@ export type TableProps<
    * @beta
    */
   enableVirtualization?: boolean;
+  /**
+   * Flag whether table has a column manager.
+   * @default false
+   */
+  hasColumnManager?: boolean;
```

Comment:
Still not sure about this name.. Currently all it does is toggles visiblity of columns. DOnt know if we gonna add something more in there..

Reply:
`enableColumnVisibility` or `enableColumnVisibilityToggle` ?

Does this mean we'd also change the name of the icon as well?

Reply:
Icon for the column manager you mean? No, I mean that icon opens menu which is supposed to be column manager (so not necessarily only visibility control, I think).
hmm.. Maybe Id vote `enableColumnVisibilityToggle` out from these 2.
But then if we find the way for the user to customize lets say that menu, then we might need a prop to just enable it to function :D

Reply:
If we plan to add column rearrange here for accessibility then I would say it can be called column manager.

Reply:
but do we plan? cause I dont think so :D

Reply:
this table prop has been removed